### ISSUE TITLE
Make NVDA read pdfs from some websites

### ISSUE THREAD

**Issue by harun-342** [nvaccess/nvda#18302]
### Is your feature request related to a problem? Please explain.
NVDA cannot read pdfs even though they are text due to accessibility issues caused by some websites.
### Explain your desired solution
It is possible to read with NVDA thanks to OCR, but make it readable without OCR.
### Explain alternatives you are considering
n/a
### Additional context

**Comment by Adriani90** [comment]
Can you please upload a sample document where we can test this?

**Comment by harun-342** [comment]
@Adriani90
My native language is Turkish, so I don't know an example in English, but what happens is that when I move with the arrows even though there is text there, NVDA says blank, not only in NVDA, but also in other screen readers.

**Comment by Adriani90** [comment]
Do you have a document for us to test?Von meinem iPhone gesendetAm 22.06.2025 um 08:00 schrieb harun-342 ***@***.***>:﻿harun-342 left a comment (nvaccess/nvda#18302)
@Adriani90
My native language is Turkish, so I don't know an example in English, but what happens is that when I move with the arrows even though there is text there, NVDA says blank, not only in NVDA, but also in other screen readers.

**Comment by harun-342** [comment]
@Adriani90
For example, it is not possible for me to read the decision on this website, which contains a decree on accessibility in my country, without applying OCR.
https://www.resmigazete.gov.tr/eskiler/2025/06/20250621-17.pdf

**Comment by davidcarlisle** [comment]
@harun-342  that PDF has no accessibility tagging. Furthermore after the heading there is no actual text in the pdf, it is some kind of screenshot so just a picture of text, OCR is the only way to extract information from it.

This does not just affect screen readers, GENELGE 2025/10 can be selected as text but then the rest of the page is an image, even though, to a sighted reader it looks like text, you can not select it or search it or interact with it in any way.

**Comment by Adriani90** [comment]
Yes, this is due to the fact that the organization which publishes this pdf file did not really care about accessibility. NVDA cannot do anything about it apart from OCR recognition.
If you open this PDF in Chrome, an automatic OCR will apply based on Gemeni AI model, but it is not 100% reliable.

I am closing as can't fix.
